specification/monitor/resource-manager/readme.java.md

```diff
@@ -0,0 +1,34 @@
+## Java
+
+These settings apply only when `--java` is specified on the command line.
+Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-libraries-for-java clone>`.
+
+``` yaml $(java)
+azure-arm: true
+fluent: true
+namespace: com.microsoft.azure.management.monitor
+license-header: MICROSOFT_MIT_NO_CODEGEN
+payload-flattening-threshold: 1
+output-folder: $(azure-libraries-for-java-folder)/azure-mgmt-monitor
+```
+
+### Java multi-api
+
+``` yaml $(java) && $(multiapi)
+batch:
+  - tag: package-2019-11
+```
+
+### Tag: package-2019-11 and java
+
+These settings apply only when `--tag=package-2019-11 --java` is specified on the command line.
+Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-sdk-for-java clone>`.
+
+``` yaml $(tag) == 'package-2019-11' && $(java) && $(multiapi)
+java:
+  namespace: com.microsoft.azure.management.monitor.v2019_11_01
+  output-folder: $(azure-libraries-for-java-folder)/sdk/monitor/mgmt-v2019_11_01
+regenerate-manager: true
+generate-interface: true
+fconfig: '{"moduleName": "Monitor"}'
```

Comment:
If no fconfig with moduleName, autorest.java will parse provider and set module name as `insights`.